FluentTc/Domain/TestOccurrences.cs

```diff
@@ -0,0 +1,15 @@
+namespace FluentTc.Domain
+{
+    public class TestOccurrences
```

Comment:
Can you please extract an interface from this class and make all its properties read-only, i.e. having only getters?